### ISSUE TITLE
Fail to draw geometry.Image using visualization.draw_geometries()

### ISSUE THREAD

**Issue by huzhlei** [intel-isl/Open3D#1678]
Runtime error [CreateCoordinateFrame] size <= 0 at
open3d.visualization.draw_geometries([open3d.geometry.**Image**])

But no problem with other Geometry types.

**To Reproduce**
In provided Open3D/examples/Python/Misc/color_image.py

y = matplotlib.image.imread("../../TestData/lena_color.jpg")
yy = o3d.geometry.Image(y)
o3d.visualization.draw_geometries([yy])

**Screenshots**
"Open3D/examples/Python/Misc/color_image.py", line 30, in <module>
    o3d.visualization.draw_geometries([yy])
RuntimeError: [Open3D ERROR] [CreateCoordinateFrame] size <= 0

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04
 - Python version: 3.6.9
 - Open3D version: 0.9.0
 - Is this remote workstation?: no
 - How did you install Open3D?: pip

**Comment by Meijia-Zhi** [comment]
hi, does your code working now , I have the same problem , but I don't know how to deal with it. Anybody help me

**Comment by huzhlei** [comment]
No. But there are many other ways to draw image, with matplotlib or opencv or etc. Maybe the authors find this draw Image feature of least importance and have removed it in current version.

**Comment by jwesolo** [comment]
I am also having this problem. I am reading the file with the native Open3D image reader:

```
an_image = o3d.io.read_image ('some_image.png')
o3d.visualization.draw_geometries([an_image])
```

I get the same error:

```
RuntimeError: [Open3D ERROR] [CreateCoordinateFrame] size <= 0
```

The same thing happens with `visualizer.run ()`:

```
visualizer = o3d.visualization.VisualizerWithEditing ()
visualizer.create_window ()
renderOptions = visualizer.get_render_option ()
renderOptions.background_color = np.asarray ([0, 0, 0])
renderOptions.point_size = 2
renderOptions.show_coordinate_frame = True
a_cloud = [<some array>]
an_image = o3d.io.read_image ('some_image.png')
visualizer.add_geometry (a_cloud)
visualizer.add_geometry (an_image)
visualizer.run ()
visualizer.destroy_window ()
```

This is the same error as https://github.com/intel-isl/Open3D/issues/1697. However, I am not trying to do anything with a mesh. I have not yet done enough reading of the source to determine if the image renderer creates a mesh to paint with the image, but I suspect that this is the case.

Thank you.

**Comment by prewettg** [comment]
This appears to be fixed in 0.12

**Comment by 459737087** [comment]
It still exist this problem in 0.14.1 version.
